course/styles/blue.json

```diff
@@ -208,7 +208,7 @@
 					"size": "3rem",
 					"fluid": {
 						"min": "3rem",
-						"max": "4rem"
+						"max": "3.25rem"
```

Comment:
Note that I'm updating the global theme style here to make things consistent. Let me know if you think this is not expected.

Reply:
I was asking @Luchadores about headings. He said that in general we consider headings are already fine. Are you fixing a specific case?
And I want to mention that now Huge is almost the same as Extra large. If we look at other sizes they have some logic in those size periods: a larger size is a continuation of a smaller.

Reply:
This font size is used for the H1 tag which is the post title.

![image](https://github.com/Automattic/themes/assets/1612178/f783448f-1489-425a-b8b8-792501cef34c)

The issue for me here is that we're using a 52px (3.25rem) font size for the H1 post title in Learning Mode and a different font size (4rem) for the H1 post title outside of Learning Mode. Is this the expectation? If this is the case, I'm not sure how to implement it because it's the same element but in a different context (learning mode vs non-learning mode). I don't know if we can do it using the JSON configs. Is there another approach?

Btw, we're having the same issue with the other variants ([gold](https://github.com/Automattic/themes/pull/7047/files#diff-d085156ba069696207d9f71dc2929662df0276144b7f4ef4e1e25b49da4efa09R196-R197), [dark](https://github.com/Automattic/themes/pull/7065/files#diff-6186ce83da9d3760d88af574107425d815430ada5b56c5920ac0f227c72c883dR171-R172)).

@Luchadores Can you help us figure this out? Is it expected for the H1 post title to be the same font size in Learning Mode and outside of it? Is it OK if we use the same font size?

Reply:
I think, a way we can address this issue is to add a rule to our Learning Mode CSS and use the variation body class for that.

Reply:
I did try using the variation body class but I've noticed that the class is removed once you make a single global style change. For example, in this case, if you change the background color, the H1 font size will change as well. Are we OK with this? For me, it's a bit unreliable.

Reply:
Hey Miro :wave-up-2: Global styles should apply everywhere. Having the same buttons, the same family-fonts are great being the same. For the font sizes, I would be careful. I believe it’s fine to have the same font size for the body of the Learning Mode content, but the course outline should stay as it is.

For example:
![image](https://github.com/Automattic/themes/assets/10038976/7f49e840-feb0-4dee-b227-9f5a331ebbd7)

Hello and welcome can be the same h1 defined in the global styles by the user. But the content on the left side would probably break if we tie up together with the same body, h1, h2, h3… font sizes.